docker-compose-cloud.yaml

```diff
@@ -0,0 +1,30 @@
+version: '3.8'
+
+services:
+  quickpizza:
+    image: ghcr.io/grafana/quickpizza-local:latest
+    ports:
+      - "3333:3333"
+    environment:
+      QUICKPIZZA_OTLP_ENDPOINT: http://agent:4318
+      QUICKPIZZA_TRUST_CLIENT_TRACEID: 1
+      # must be set with an .env file
```

Comment:
Users must configure env variables in an .env file. 

I'll add the proper instructions on the readme if this is approved.

Reply:
See -> https://github.com/grafana/quickpizza/tree/pepe/docs/grafana-agent#grafana-cloud